packages/teams-js/src/private/store.ts

```diff
@@ -150,6 +162,7 @@ export async function openStoreExperience(openStoreParams: OpenStoreParams): Pro
       openStoreParams.dialogType,
       (openStoreParams as OpenAppDetailParams).appId,
       (openStoreParams as OpenSpecificStoreParams).collectionId,
+      JSON.stringify(openStoreParams.size),
```

Comment:
There should be new unit tests for this change

Reply:
added